misc/services/mobile-webui/mobile-webui-frontend/src/reducers/wfProcesses/index.js

```diff
@@ -104,11 +104,17 @@ export const getStepByIdFromLine = (line, stepId) => {
   return line?.steps?.[stepId];
 };
 
-export const getStepByQRCodeFromActivity = (activity, lineId, qrCode) => {
+export const getNonIssuedStepByQRCodeFromActivity = (activity, lineId, qrCode) => {
```

Comment:
aren't these functions specific to manufacturing?
if that's the case, I would suggest to move them somewhere in the manufacturing related code.

Reply:
I guess those was here before we had the different places target per manufacturing/picking and so on, will check 👍